chart/openfaas/templates/psp.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.psp }}
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+    name: {{ .Release.Name }}-psp
+    labels:
+        app: {{ template "openfaas.name" . }}
+        chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+    annotations:
+        seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+spec:
+    privileged: false
+    hostIPC: false
+    hostNetwork: false
+    hostPID: false
+    readOnlyRootFilesystem: false
+    allowPrivilegeEscalation: false
+    allowedCapabilities:
+        - '*'
+    fsGroup:
+        rule: RunAsAny
+    hostPorts:
```

Comment:
Is this a good idea?

Reply:
hostNetwork is disabled so those ports will never be used